portfolio/src/main/webapp/myMap.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Map</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAXHeiUib7BmE7tup3E0FWsn41k1lJMuFs"></script>
+    <script>
```

Comment:
We typically try to avoid putting js directly in the html, it makes it extremely difficult to test and debug, can you put this in the script.js file. You may need to add some checks to see if it's the myMap page that's being loaded.